firmware/klipper_configurations/SKR_1.3/Voron2_SKR_13_Config.cfg

```diff
@@ -275,11 +277,11 @@ dir_pin: P0.11
 enable_pin: !P2.12
 ##	16 microsteps Mobius 3 ~= 0.00180
 ##	Update value below when you perform extruder calibration
-##	Higher value means less filament extruded
 ##	If you ask for 100mm of filament, but in reality it is 98mm:
-##	step_distance = 98 / 100 * step_distance_old
-##  0.00240 a good starting value for Afterburner, 0.00180 for Mobius
-step_distance: 0.00240
+##	rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+##  7.68 a good starting value for Afterburner, 5.76 for Mobius
+rotation_distance: 7.68
```

Comment:
have you validated this number? This is the one area that I'm not 100% on

Reply:
It's taken straight from the formula in the klipper docs, but yes, I have tested it.

It is very very close to accurate on my V2.